pkgs/development/python-modules/django-storages/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildPythonPackage, fetchPypi
+, django
+}:
+
+buildPythonPackage rec {
+  pname = "django-storages";
+  version = "1.7.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "01xq232h321716r08rari9payas7fsiwwr5q6zgcrlwkckwxxczk";
+  };
+
+  buildInputs = [ django ];
```

Comment:
This should be `propagateBuildInputs`
* https://github.com/jschneier/django-storages/blob/master/setup.py#L16